erpnext/regional/doctype/fta_audit_file/fta_audit_file.py

```
@@ -0,0 +1,659 @@
+			self.generation_log = result["log"]
+			self.status = "Generated"
+			self.save()
+		except Exception as e:
+			try:
+				err_doc = frappe.get_doc(self.doctype, self.name)
+				err_doc.status = "Error"
+				err_doc.error_message = str(e)
+				err_doc.generation_log = (err_doc.generation_log or "") + f"\n\nError: {e}"
+				err_doc.save()
+			except Exception:
+				pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)